test/runtests.jl

```diff
@@ -151,8 +151,8 @@ jagmesh_adj = jagmesh()
           for i in 1:size(a,1)
              p = Int32[]
              for e in collect(edges(g))
-                 if e[1] == i
-                     push!(p,e[2])
+                 if e.src == i
```

Comment:
This should be `src(e)` for LightGraphs - use the accessor function, not the internal field.

Same with L155 - `dst(e)`.